### PR TITLE
CASSANDRA-17699 Updated broken link in development/documentation.html

### PR DESCRIPTION
**Pull request by msacks-dev**
i'm not sure if this is correct, please review.

### CODE REVIEW

**Review by msacks-dev — COMMENTED** [review]
i probably should have built this before pull requesting huh?

File: site-content/source/modules/ROOT/pages/development/documentation.adoc [hunk 1]
```diff
@@ -19,7 +19,7 @@ instead of visual editors (which some people would call good news).
 Antora processes http://www.asciidoc.org[Ascidoc], the markup language used to generate our documentation.
 Markup languages allow you to format text using certain syntax elements.
 Your document structure will also have to follow specific conventions.
-Feel free to take a look at http://cassandra.apache.org/docs[existing documents] to get a better idea how we structure our documents.
+Feel free to take a look at link:/doc/latest/[existing documents] to get a better idea how we structure our documents.
```
Comment (michaelsembwever) [inline site-content/source/modules/ROOT/pages/development/documentation.adoc]: i don't think this is intended to go to /doc/latest/ but /doc/ but it's supposed to show you the different versions we have.

e.g. see https://cassandra.apache.org/doc/

**Comment by michaelsembwever** [comment]
> i probably should have built this before pull requesting huh? 

ideally yes. 

(we are also very keen for someone to contribute a `.github/workflows/website.yml` that does 
```
            ./run.sh website-ui bundle -a BUILD_USER_ARG:`whoami` -a UID_ARG:`id -u` -a GID_ARG:`id -g` ;
            ./run.sh website container -a BUILD_USER_ARG:`whoami` -a UID_ARG:`id -u` -a GID_ARG:`id -g` ;
            ./run.sh website build -g ;
```
(ref: https://github.com/apache/cassandra-builds/blob/trunk/jenkins-dsl/cassandra_job_dsl_seed.groovy#L1387-L1393 )
if i can tempt you ;) )

File: site-content/source/modules/ROOT/pages/development/documentation.adoc (hunk 1 above)
Comment (msacks-dev) [reply site-content/source/modules/ROOT/pages/development/documentation.adoc]: So drop /latest/?

Comment (msacks-dev) [reply site-content/source/modules/ROOT/pages/development/documentation.adoc]: willdo

**Comment by ErickRamirezAU** [comment]
> i probably should have built this before pull requesting huh?

Yes, you should always build + stage locally to make sure that your updates render as expected, links get generated by asciidoc, etc. 🙂

Also, please post a screenshot of the page in the ticket to show that you've staged the changes and that you've verified it locally. See CASSANDRA-17703 if you need an example. Cheers! 🍻

File: site-content/source/modules/ROOT/pages/development/documentation.adoc [hunk 2]
```diff
@@ -19,7 +19,7 @@ instead of visual editors (which some people would call good news).
 Antora processes http://www.asciidoc.org[Ascidoc], the markup language used to generate our documentation.
```
Comment (ErickRamirezAU) [inline site-content/source/modules/ROOT/pages/development/documentation.adoc]: If you recall in the ticket, I've also requested you to fix the typo. 🙏
```suggestion
Antora processes http://www.asciidoc.org[AsciiDoc], the markup language used to generate our documentation.
```

**Review by ErickRamirezAU — CHANGES_REQUESTED** [review]

File: site-content/source/modules/ROOT/pages/development/documentation.adoc (hunk 1 above)
Comment (msacks-dev) [reply site-content/source/modules/ROOT/pages/development/documentation.adoc]: done

**Comment by msacks-dev** [comment]
Done but I still need time to figure out the build process so submitted without testing tbh. Should work cleanly though.

**Comment by msacks-dev** [comment]
@michaelsembwever @ErickRamirezAU Let me know whats left to get this accepted. So I run the commands in the README to build basically, test, and then do I have to open a new pull request or can I use this one? All guidance appreciated. 
Also I'd be happy to look into the request to use the https://github.com/apache/cassandra-builds/blob/trunk/jenkins-dsl/cassandra_job_dsl_seed.groovy#L1387-L1393 method but just have some questions .

**Comment by ErickRamirezAU** [comment]
> Done but I still need time to figure out the build process so submitted without testing tbh. Should work cleanly though.

Last week, I sent you the 3 simple steps to stage the website locally. Here is the [Slack thread](https://the-asf.slack.com/archives/C01RY1LUPD5/p1655439069291909). 🙂

**Comment by msacks-dev** [comment]
Re-reading slack thread fyi.

**Comment by michaelsembwever** [comment]
hi @msacks-dev , any progress with this? reach out if you have any questions or would like any help! we very much appreciate all the time you spend on the contribution :)

**Comment by msacks-dev** [comment]
Please remind me what needs to be done on my end to get it integrated?

**Comment by michaelsembwever** [comment]
@ErickRamirezAU , was there a specific change being requested? or just the request to run `./run.sh website build` and take a snapshot of the new page for verification? 

@msacks-dev , I think just `./run.sh website build` is enough now, as opposed to our previous advice of having to run multiple steps…

**Comment by msacks-dev** [comment]
Got it. On the road but will do this evening. Thanks for the reminder!

**Comment by Msacks-stonefish** [comment]
Is it ok if I tackle this Thursday?


Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows

**Comment by michaelsembwever** [comment]
> Is it ok if I tackle this Thursday?

yes ofc @msacks-dev , we're OSS –  patience and context-switching is what we do best 😎

**Comment by msacks-dev** [comment]
I appreciate it!

**Comment by Msacks-stonefish** [comment]
Setting up VirtualBox/Linux now to test changes in my PR (because Docker on Windows 11 doesn’t seem to be all there yet).

I’ll update if I have any troubles.

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows

From: Matthew ***@***.***>
Sent: Sunday, July 17, 2022 8:15 PM
To: ***@***.***>; ***@***.***>
Cc: ***@***.***>
Subject: RE: [apache/cassandra-website] CASSANDRA-17699 Updated broken link in development/documentation.html (PR #143)

Is it ok if I tackle this Thursday?


Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows

**Review by msacks-dev — COMMENTED** [review]
I think this can be rejected/closed as the link didn't work when I built it.  See PR 157 for updated fix.

**Comment by michaelsembwever** [comment]
Superseded by https://github.com/apache/cassandra-website/pull/157

**Pull request closed without merging**